frontend/test/metabase/scenarios/filters/reproductions/25994-between-after-summarize-not-working.cy.spec.js

```diff
@@ -0,0 +1,46 @@
+import {
+  restore,
+  visitQuestionAdhoc,
+  popover,
+  visualize,
+} from "__support__/e2e/helpers";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATABASE;
+
+const questionDetails = {
+  dataset_query: {
+    type: "query",
+    query: {
+      "source-table": PRODUCTS_ID,
+      aggregation: [
+        ["min", ["field", PRODUCTS.CREATED_AT, { "temporal-unit": "day" }]],
+      ],
+      breakout: [["field", PRODUCTS.CATEGORY, null]],
+    },
+    database: 1,
```

Comment:
ID or doesn't it default to db 1 already?
```suggestion
    database: SAMPLE_DB_ID,
```

Reply:
Almost missed it.
Thanks.